modules/admin_manual/pages/configuration/files/default_files_configuration.adoc

```diff
@@ -20,7 +20,11 @@ image:skeleton-files1.png[image]
 == Additional Configuration
 
 The configuration option `skeletondirectory` available in your `config.php` allows you to configure the directory where the skeleton files are located.
+
 These files will be copied to the data directory of new users.
-Leave empty to not copy any skeleton files.
+
+Leave the `/core/skeleton` directory empty if you do not want to copy any skeleton files.
```

Comment:
Just say "Leave this directory empty" - as you can theoretically also define a custom path apart from `/core/skeleton`

Reply:
changed